test-projects/basic/cypress/integration/add-edit-delete-data.js

```diff
@@ -54,6 +54,7 @@ describe('Adding data', () => {
     cy.get('#ks-input-author').click({ force: true });
     cy.get('#ks-input-author').type('John Doe{downarrow}{enter}', {
       force: true,
+      delay: 100,
```

Comment:
Is this change still required for the tests to work?

Reply:
Yes, it now needed a short delay after typing for the results to appear in the dropdown, else it would false.